app/jobs/globus_setup_job.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+# Handles Globus setup actions (create endpoints if possible, send emails)
+class GlobusSetupJob < ApplicationJob
+  queue_as :default
+
+  # rubocop:disable Metrics/AbcSize
+  def perform(work_version)
+    druid = work_version.work.druid # may be nil
+    user = work_version.work.owner
+    Honeybadger.context({ work_version_id: work_version.id, druid:,
+                          work_id: work_version.work.id, depositor_sunet: user.sunetid })
+
+    if globus_user_exists?(user.email) && work_version.globus_endpoint.blank?
+      # user is known to globus but doesn't have a globus endpoint yet, so create it and send the email
+      create_globus_endpoint(work_version)
+      WorkObserver.globus_endpoint_created(work_version) # send email with globus endpoint
```

Comment:
Seems odd that this is being invoked through WorkObserver. Typically, WorkObserver is only invoked via the state machine.

Reply:
removed `WorkObserver` method and send email directly now from the job